Documentation/manual-install.md

```diff
@@ -4,6 +4,17 @@ If you want to install metering without OLM, using what's currently in master, f
 Next, decide which namespace you want to install Metering into, and set the `METERING_NAMESPACE` environment variable to the namespace you want to use.
 By default, if it's unset, it will use the `metering` namespace.
 
+## Requirements
+
+Our installation scripts are written in bash, and utilize a few non-standard tools to interact with yaml and json files.
+Please ensure you have the following tools installed before running the install scripts:
+
+- bash
+- [faq](https://github.com/jzelinskie/faq) 0.0.5 or newer
+  - For Fedora 29, Rawhide, and CentOS 7 you can use the following copr repo: https://copr.fedorainfracloud.org/coprs/ecnahc515/faq/
+  - For mac: `brew install faq`
```

Comment:
Superminor but on Mac it was, `brew tap jzelinskie/faq and then brew install faq` however I do see up on line 13 the repo itself of a dependency.

Reply:
Good catch, I'll fix.